crates/ef-testing/src/evm_sequencer/evm_state.rs

```diff
@@ -191,6 +197,50 @@ impl EvmState for KakarotSequencer {
             Into::<FieldElement>::into(nonce).to_bytes_be(),
         ))
     }
+
+    fn get_code_at(&mut self, evm_address: &Address) -> StateResult<Bytes> {
```

Comment:
how can we know this works?

Reply:
Running `cargo test` runs the add.json and mul.json files from the ef-tests folder in the VMTests suite. If this wouldn't work, we would have issues there.